ligand/templates/alignment.html

```diff
@@ -0,0 +1,349 @@
+{% extends "home/base.html" %}
```

Comment:
What is this template doing here? It seems like a copy of the seq. alignment template which is not used here in the ligand app?

Reply:
True; deleted